x-pack/plugins/security_solution/public/timelines/components/timeline/body/actions/action_icon_item.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiButtonIcon, EuiToolTip } from '@elastic/eui';
+
+import { EventsTd, EventsTdContent } from '../../styles';
+import { DEFAULT_ICON_BUTTON_WIDTH } from '../../helpers';
+
+interface ActionIconItemProps {
+  ariaLabel?: string;
+  id: string;
+  width?: number;
+  dataTestSubj?: string;
+  content?: string;
+  iconType?: string;
+  isDisabled?: boolean;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
the linter can be re-enabled by changing the import of `React` to include the following:

```
import React, { MouseEvent } from 'react';
```

and changing

```
onClick?: any;
```

to

```
onClick?: (event: MouseEvent) => void;
```

That passes the type checker:

```
➜  kibana git:(feat/alert-actions-timeline) ✗ node scripts/type_check.js --project x-pack/tsconfig.json
 ✔ x-pack
```